addons/mail/static/src/js/extended_chat_window.js

```diff
@@ -44,6 +49,14 @@ return ChatWindow.extend({
     on_keydown: function (event) {
         event.stopPropagation();
     },
+    on_cilck_expand: function () {
+        event.preventDefault();
+        chat_manager.is_ready.then(this.discuss_expand.bind(this));
+    },
+    discuss_expand: _.debounce(function () {
+        core.bus.trigger('change_menu_section', chat_manager.get_discuss_menu_id());
+        this.do_action('mail.mail_channel_action_client_chat', {clear_breadcrumbs: true, active_id: this.channel_id});
+    }, 1000, true),
```

Comment:
Only trigger 'change_menu_section' once the action has been loaded to prevent flickering (like it is done [here](https://github.com/odoo/odoo/blob/10.0/addons/mail/static/src/js/systray.js#L36)).